test/integration/apiserver/apiserver_test.go

```diff
@@ -527,12 +527,6 @@ func testListOptionsCase(t *testing.T, rsClient appsv1.ReplicaSetInterface, watc
 		}
 		return
 	}
-	if opts.ResourceVersion == invalidResourceVersion {
```

Comment:
I don't understand.. why is this move necessary?

Reply:
the old code checks validity of opts.ResourceVersion before continuation, the new code checks the validity of continuation first. The order difference would result in different err message for the test case of continuation along with an invalid opts.rv